archive_viewer/main.py

```diff
@@ -118,30 +118,62 @@ def setup_ui(self):
         range_layout.addWidget(keep_range_label)
         range_layout.addWidget(type_lable, 1, 2)
 
-        # time Menu
-        min_label_time = QLabel("Start:")
-        max_label_time = QLabel("End:")
-        min_input_time = QLineEdit()
-        max_input_time = QLineEdit()
-        keep_range_label_time = QLabel("Keep Ranges")
-        keep_range_check_box_time = QCheckBox()
-        type_label_time = QLabel("Type")
-
-
-        time_tab = QWidget()
-        time_layout = QGridLayout()
-        time_layout.addWidget(min_label_time)
-        time_layout.addWidget(min_input_time,  0, 1)
-        time_layout.addWidget(max_label_time)
-        time_layout.addWidget(max_input_time,  1, 1)
-        time_layout.addWidget(keep_range_check_box_time)
-        time_layout.addWidget(keep_range_label_time)
-        time_layout.addWidget(type_label_time, 1, 2)
+        # Create the time axis table widget
+        time_axis_table = QTableWidget()
+        time_axis_table.setColumnCount(6)
+        time_axis_table.setHorizontalHeaderLabels(["Axis Name", "Start", "End", "Calendar", "Slider", "Position"])
+        
+        # Add a row to the table
+        time_axis_table.insertRow(0)
+        
+        # Populate the cells with widgets and data
+        axis_name_item = QTableWidgetItem("Time Axis 1")
+        time_axis_table.setItem(0, 0, axis_name_item)
+        
+        start_item = QTableWidgetItem("2023-06-21")
```

Comment:
is it possible to not have a default date or maybe to have the current date set as the default?